### DISCUSSION TITLE
Point where everything initializes and strapi.config is populated

### DISCUSSION THREAD

**Discussion by cenap** [strapi/strapi#7487] [Legacy Topics]
I give "yarn develop" command and everything initializes smoothly. I need to run some custom code including **"strapi.config"** somewhere after eveything initializes. If I try to write my code in **bootstrap.js**, **"strapi.config"** is empty since the initialization is not completed yet.

Where is the right place to add my code to get values from strapi.config?

**Comment by derrickmehaffy** [comment]
Hi @cenap can you clarify which version of Strapi you are on? I just tested on the latest release (v3.1.4) and was able to `console.log(strapi.config)` within the bootstrap.

**Comment by cenap** [comment]
I am on the latest release indeed. When I check I see that I can also get
strapi.config which means it is not empty and I was wrong.

Thank you for the answer...

DMehaffy <notifications@github.com>, 17 Ağu 2020 Pzt, 17:49 tarihinde şunu
yazdı:
